### PR TITLE
[Don't Merge] Docs for 2.3

### PR DESCRIPTION
**Pull request by shavidissa**

### CODE REVIEW

File: configuring/configuring-healthwatch.html.md.erb [hunk 1]
```diff
@@ -529,6 +531,24 @@ To configure the **Resource Config** pane:
 
 1. Click **Save**.
 
+## <a id='resource-config'></a> (Optional) Configure for OpenTelemetry
+
+Follow these steps to configure HealthWatch and receive data through the OpenTelemetry collector for Tanzu Application Service 6.0 and above:
+
+1. Click the Settings tab and select **Resource Config**.
```
Comment (dubeyshefali) [inline configuring/configuring-healthwatch.html.md.erb:538]: Click on Healthwatch exporter tile and select **Resource Config** under the Settings tab.

File: configuring/configuring-healthwatch.html.md.erb [hunk 2]
```diff
@@ -529,6 +531,24 @@ To configure the **Resource Config** pane:
...
+    1. Set the **TAS Configure Exporter** and **TAS Gauge Exporter** configurations to `O` because they are only used by FireHose.
+    1. If you made any changes, click **Save**.
```
Comment (dubeyshefali) [inline configuring/configuring-healthwatch.html.md.erb:540]: Click **Save**

File: configuring/configuring-healthwatch.html.md.erb [hunk 3]
```diff
@@ -529,6 +531,24 @@ To configure the **Resource Config** pane:
...
+1. Click the **Certificates** tab:
```
Comment (dubeyshefali) [inline configuring/configuring-healthwatch.html.md.erb:541]: **Credentials**

File: configuring/configuring-healthwatch.html.md.erb [hunk 4]
```diff
@@ -529,6 +531,24 @@ To configure the **Resource Config** pane:
...
+    1. Click **Link to Credential** next to **Healthwatch Exporter Client Mtls**.
+    1. Copy the certificate content and use it for step 3.2.1 below.
+    1. If you made any changes, click **Save**.
+1. Open the Tanzu Application Service tile.
+      1. Click Settings and select **System Logging**.
+        1. Deselect the **Enable V1 Firehose** and **Enable V1 Firehose** configurations.
```
Comment (dubeyshefali) [inline configuring/configuring-healthwatch.html.md.erb:547]: Unselect

File: configuring/configuring-healthwatch.html.md.erb [hunk 5]
```diff
@@ -529,6 +531,24 @@ To configure the **Resource Config** pane:
...
 
+## <a id='resource-config'></a> (Optional) Configure for OpenTelemetry
+
+Follow these steps to configure HealthWatch and receive data through the OpenTelemetry collector for Tanzu Application Service 6.0 and above:
+
+1. Click the Settings tab and select **Resource Config**.
+    1. Set the **TAS Configure Exporter** and **TAS Gauge Exporter** configurations to `O` because they are only used by FireHose.
+    1. If you made any changes, click **Save**.
+1. Click the **Certificates** tab:
+    1. Click **Link to Credential** next to **Healthwatch Exporter Client Mtls**.
```
Comment (dubeyshefali) [inline configuring/configuring-healthwatch.html.md.erb:542]: This will change based on the new config that we add for certificates

**Review by dubeyshefali — COMMENTED** [review]
Few minor comments. Looks good otherwise.

File: configuring/configuring-healthwatch.html.md.erb (hunk 5 above)
Comment (shavidissa) [reply configuring/configuring-healthwatch.html.md.erb:542]: Thank you! Please let me know once it is decided.

**Pull request merged**